charts/stable/lychee/values.yaml

```
@@ -6,76 +6,49 @@
 #
 
 image:
+  # -- image repository
   repository: lycheeorg/lychee-laravel
+  # -- image tag
+  tag: v4.3.0
+  # -- image pull policy
   pullPolicy: IfNotPresent
-  tag: v4.0.8
 
-strategy:
-  type: Recreate
 
-env: {}
-  # PHP_TZ: UTC
-  # PUID: 1001
-  # PGID: 1001
+# -- environment variables.
+# @default -- See below
+env:
+  # -- Set the container timezone
+  PHP_TZ: UTC
+    # -- Specify the user ID the application will run as
```

Looks like there's an extra space.
```suggestion
   # -- Specify the user ID the application will run as
```